modules/apps/object/object-test/src/testFunctional/macros/ObjectAdmin.macro

```diff
@@ -837,21 +848,29 @@ definition {
 			Click(
 				key_type = "${thenAction}",
 				locator1 = "ObjectAdmin#SELECT_RELATIONSHIPS_TYPE_OPTION");
-		}
 
-		if (isSet(onAction)) {
-			Click(locator1 = "ObjectAdmin#ENTRY_CHOOSE_OPTION");
+			if ("${thenAction}" == "Add an Object Entry") {
+				Click(locator1 = "ObjectAdmin#CHOOSE_AN_OBJECT");
 
-			Select(
-				chooseOption = "${chooseOption}",
-				locator1 = "ObjectAdmin#SELECT_CHOOSE_OPTION");
-		}
+				ClickNoError.clickNoMouseOver(
+					key_objectLabel = "${objectLabel}",
+					locator1 = "ObjectAdmin#CHOOSE_OBJECT_LABEL");
 
-		if (isSet(inputAction)) {
-			if ("${inputActionActive}" == "true") {
 				Click(
-					locator1 = "ObjectAdmin#INPUT_METHOD",
-					value1 = "${inputMethod}");
```

Comment:
What happens to tests that are setting `onAction` and `inputAction`?

Reply:
There is no test that set this